test/e2e/volume_create_test.go

```diff
@@ -79,6 +79,39 @@ var _ = Describe("Podman volume create", func() {
 		Expect(check.OutputToString()).To(ContainSubstring("hello"))
 	})
 
+	It("podman create and import volume", func() {
+		if podmanTest.RemoteTest {
+			Skip("Volume export check does not work with a remote client")
+		}
+
+		session := podmanTest.Podman([]string{"volume", "create", "my_vol"})
+		session.WaitWithDefaultTimeout()
+		volName := session.OutputToString()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"run", "--volume", volName + ":/data", ALPINE, "sh", "-c", "echo hello >> " + "/data/test"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"volume", "export", volName, "--output=hello.tar"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"volume", "create", "my_vol2"})
+		session.WaitWithDefaultTimeout()
+		volName = session.OutputToString()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"volume", "import", "my_vol2", "hello.tar"})
```

Comment:
Could you add a system tests, that tests stdin?

Reply:
sure.

Reply:
@rhatdan this is added in latest commit.